AppCenterPush/AppCenterPush/Internal/DelegateForwarder/MSUserNotificationCenterDelegateForwarder.m

```diff
@@ -23,7 +23,7 @@ + (void)load {
   // TODO test the forwarder on macOS.
   // Register selectors to swizzle (iOS 10+).
 #if !TARGET_OS_OSX
-  if ([[MSUserNotificationCenterDelegateForwarder sharedInstance] originalClassForSetDelegate]) {
+  if (@available(iOS 10.0, tvOS 10.0, watchOS 3.0, *)) {
```

Comment:
Is there a way to exclude `macOS` instead of listing all versions for platforms? Those versions exactly match with our minimum supported version.

Reply:
As I know the minimal version for iOS is 9.0
https://docs.microsoft.com/en-us/appcenter/sdk/getting-started/ios

s.ios.deployment_target = '9.0'
https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/AppCenter.podspec

Reply:
Sorry for confusion. I'm getting confused this a lot of time.